app/dashboard/templates/onepager/send2.html

```diff
@@ -172,7 +172,7 @@ <h3>{% trans "with instructions about how to receive their tip." %}</h1>
   {% if not user_json and username %}
     <script>
     setTimeout(function() {
-      alert("Sorry, we can find the user @{{ username }}");
+      alert("Sorry, we can find the user");
```

Comment:
@thelostone-mc we should use `_alert` here, right?

Reply:
done!